src-built-in/components/assets/css/_windowTitleBar.css

```diff
@@ -452,4 +407,54 @@ html.desktop-active .fsbl-linker[hover=true] {
     position: relative;
     height: 100%;
     left: 0px;
+}
+
+
+@media screen and (min-width: 0px) and (max-width: 345px){
```

Comment:
@carterbs need a comment above each of these media tags explaining the situation it is dealing with.